storybook/stories/core/components/List2/List2.stories.js

```diff
@@ -136,3 +137,37 @@ NoSidepanel.parameters = {
     },
   },
 };
+
```

Comment:
The props, slots, and events for the list component are not documented fully.  Do we only have showCollapse and flatTop props there?

Reply:
Missing props added. only 3 props are there and no events